From ab38dfbf0c7fc1b3ee3234729155238c7ddf973c Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Fri, 31 Jul 2020 14:32:15 +0800
Subject: [PATCH] audio: scx15: Temporarily disable low_power settings for now.

It seems to cause 3x more cpu usage for some unknown reason.

Change-Id: I917925352937676334ed376b563433a2832fc1ff
Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
---
 audio/scx15/audio_hw.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/audio/scx15/audio_hw.c b/audio/scx15/audio_hw.c
index eedc0b1..72e8f36 100644
--- a/audio/scx15/audio_hw.c
+++ b/audio/scx15/audio_hw.c
@@ -2222,12 +2222,15 @@ static ssize_t out_write(struct audio_stream_out *stream, const void* buffer,
         out_frames = RESAMPLER_BUFFER_SIZE / frame_size;
 
         if ((low_power != out->low_power) || (out->write_threshold == -1)) {
+#if 0
             if (low_power) {
                 out->config.avail_min = LONG_PERIOD_SIZE * PLAYBACK_LONG_PERIOD_COUNT - in_frames + VBC_BASE_FRAME_COUNT;
                 out->write_threshold = out->config.avail_min;
                 ALOGW("low_power out->write_threshold=%d, config.avail_min=%d, start_threshold=%d",
                         out->write_threshold,out->config.avail_min, out->config.start_threshold);
-            } else {
+            } else
+#endif
+            {
                 out->config.avail_min = SHORT_PERIOD_SIZE / 2;
                 out->write_threshold = SHORT_PERIOD_SIZE * (PLAYBACK_SHORT_PERIOD_COUNT);
                 ALOGW("out->write_threshold=%d, config.avail_min=%d, start_threshold=%d",
-- 
2.17.1

